publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/service/MessageServiceRMQImpl.java

```diff
@@ -73,6 +78,19 @@ public SendResult send(Map<String, String> routingKeyMap, Map<String, String> ms
                             PropertiesConfig.SERVER_DOWN_MESSAGE);
                     checkEventStatus = false;
                 }
+                } catch (NackException e) {
+                    event = new PublishResultItem(entry.getKey(), 500, PropertiesConfig.SERVER_DOWN,
```

Comment:
500 should be replace by HttpStatus.SC_INTERNAL_SERVER_ERROR. The same for 504: SC_GATEWAY_TIMEOUT.

Reply:
done